### ISSUE TITLE
/root/.packages/builtin_files/geany is empty if Geany is in adrv

### ISSUE THREAD

**Issue by dimkr** [puppylinux-woof-CE/woof-CE#2421]
This is some extra weirdness in builtin_files.sh. In dpup, Geany is in the adrv, so the loop in builtin_files.sh doesn't find its files.

Is this the desired behavior? It's impossible to remove Geany and keep the rest of adrv.

See the comments in #2417.

**Comment by rizalmart** [comment]
@dimkr  I suggest to rewrite it from scratch and make /var/packages/builtin_files modular by including /var/packages/builtin_files and /var/packages/package-specs on every creation of SFS modules. Take a look at the SFS modules of QuickPup

**Comment by dimkr** [comment]
@rizalmart I agree the current design of this builtin file listing feature is not ideal, but my policy with anything that's part of PPM, or related to PPM, is to change as little as possible.
